core/services/relay/dydx/config_digester.go

```diff
@@ -0,0 +1,48 @@
+package dydx
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+const ConfigDigestPrefixDydx types.ConfigDigestPrefix = 4
+
+var _ types.OffchainConfigDigester = (*OffchainConfigDigester)(nil)
+
+type OffchainConfigDigester struct {
+	// endpoint type can be staging or prod
+	endpointType string
```

Comment:
Isn't there some more information here?
E.g. how do we disambiguate different price feeds? What if there are multiple startknet instances being targeted?

Reply:
Yes, added now.